Update_user.css

```diff
@@ -1,16 +1,23 @@
+/* Import  font from google fonts*/
```

Comment:
I import font from google fonts. The fonts I choice is Source sans pro and Open sans